### PR TITLE
Create the 2.12 branches

### PR DESCRIPTION
**Pull request by mprahl**
This includes a few fixes.

### CODE REVIEW

File: CURRENT_SUPPORTED_VERSIONS [hunk 1]
```diff
@@ -1,3 +1,4 @@
+2.11
 2.10
 2.9
 2.8
```
Comment (dhaiducek) [inline CURRENT_SUPPORTED_VERSIONS:4]: Keeping 2.8 around is right, but doesn't really _feel_ right 😕

**Review by dhaiducek — APPROVED** [review]
Thanks for the updates! I've held it until the Prow PRs merge, since we'll want those configurations in place before the branches are created.

File: CURRENT_SUPPORTED_VERSIONS (hunk 1 above)
Comment (mprahl) [reply CURRENT_SUPPORTED_VERSIONS:4]: Yeah, the script removed it I believe but it needs to stay a little while longer.

**Comment by mprahl** [comment]
@dhaiducek I added the Policy Generator to `repos.txt`.

**Review by dhaiducek — APPROVED** [review]
/unhold

**Comment by dhaiducek** [comment]
Let's do this! 😄

**Pull request merged**